### ISSUE TITLE
Ubuntu Bionic rollout to jumpbox host(s)

### ISSUE THREAD

**Issue by adborden** [GSA/data.gov#712]
## User Story
As an operator, I need new Ubuntu Bionic hosts provisioned in all of our environments so that I can continue receiving security updates automatically from Ubuntu.

Part of https://github.com/GSA/datagov-deploy/issues/571

Ubuntu 14.04 support ends [April 2019](https://www.ubuntu.com/about/release-cycle). We should move to Ubuntu 16.04 or 18.04. The latest 18.04 is preferred, but we should consider what kind of support exists for it.

## Details
Hosts are provisioned in these environments and play books are successfully applied.
1. [x] bionic sandbox
1. [x] staging
1. [x] production

## Acceptance Criteria
- [x] Bionic hosts are provisioned and operating correctly
- [x] Bionic hosts reporting to our monitoring systems
- [x] @JJediny can connect to the jumpbox and run playbooks
- [x] Old Trusty hosts are decommissioned and removed from Ansible inventory

## Definition of Done

The above acceptance criteria must be met.

**Comment by adborden** [comment]
Hosts are provisioned, waiting on [Firewall Change Request](https://gsa.service-now.com/sp/?sys_id=3dad4158db3c7fc40e1d368d7c9619e0&view=sp&id=ticket&table=sc_req_item).

**Comment by adborden** [comment]
SecOps says the trendmicro rules have been pushed, but we still can't connect to crm-web1d and dashboard-web2d. I will continue with the provisioning, but we can't remove jump1d until this is resolved.

**Comment by adborden** [comment]
The firewall issues are resolved.

Everything seems to be working correctly, except for the trendmicro role. The .deb package works for Ubuntu 14.04 only. I've requested an updated version for Ubuntu 18.04 from SecOps. They have to reach out to the vendor.

**Comment by adborden** [comment]
The firewall issue is resolved.

When running the common role, I get an [error installing the trendmicro.deb](https://github.com/GSA/datagov-deploy-common/issues/15). We're waiting on an updated .deb from secops.

**Comment by adborden** [comment]
Trendmicro issue is resolved.

@JJediny is working on getting [filebeat](https://github.com/GSA/datagov-deploy-common/issues/22) running properly.

**Comment by adborden** [comment]
Filebeat issue is [unblocked](https://github.com/GSA/datagov-deploy/pull/780), although there is [another issue in the role](https://github.com/GSA/datagov-deploy-common/issues/24). I am avoiding it with `--skip-tags filebeat`.

**Comment by adborden** [comment]
@JJediny I think we are ready to remove the Trusty hosts. Please log into jump2p/jump2d and make sure you have everything you need.

**Comment by adborden** [comment]
Confirmed with @JJediny that instances are 👍 

Submitted [REQ0445288](https://gsa.service-now.com/sp/?id=sc_request&table=sc_request&sys_id=3aa7dcd3db9933843068fd0d0f961966) to remove the old instances.
